packages/dev/create/bin/create.js

```diff
@@ -0,0 +1,144 @@
+/**
+ * @license
+ * Copyright 2020 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @fileoverview A script for creating Blockly packages based on pre-existing
+ * templates.
+ * @author samelh@google.com (Sam El-Husseini)
+ */
+
+'use strict';
+
+const chalk = require('chalk');
+const args = process.argv.slice(2);
+const fs = require('fs-extra');
+const path = require('path');
+const execSync = require('child_process').execSync;
+
+const root = process.cwd();
+
+const usage = `
+  ${chalk.blue('blockly-scripts create')}\
+  ${chalk.green('<plugin|field|block>')}\
+  ${chalk.green('<package-directory>')}
+
+For example:
+  ${chalk.blue('blockly-scripts create')}\
+  ${chalk.green('plugin')} ${chalk.green('my-blockly-plugin')}\n`;
+
+const packageType = args[0];
+// Check package type exists.
+if (!packageType) {
+  console.error('Please specify the package type.');
+  console.log(`It can either be ${chalk.green('plugin')},\
+  ${chalk.green('field')} or ${chalk.green('block')}.`);
+  console.log(usage);
+  process.exit(1);
+}
+// Check package type.
+if (!['plugin', 'field', 'block'].includes(packageType)) {
+  console.error(`Unknown package type: ${chalk.red(packageType)}`);
+  console.log(usage);
+  process.exit(1);
+}
+const packageName = args[1];
+// Check package name.
+if (!packageName) {
+  console.error('Please specify the package directory:');
+  console.log(usage);
+  process.exit(1);
+}
+const packageDir = path.join(root, packageName);
+// Check package name directory doesn't already exist.
+if (fs.existsSync(packageDir)) {
+  console.error(`Package directory already exists,
+    Remove ${packageName} and try again.`);
+  process.exit(1);
+}
+
+console.log(`Creating a new Blockly\
+  ${chalk.green(packageType)} in ${chalk.green(root)}.\n`);
+
+// Create the package directory.
+fs.mkdirSync(packageDir);
```

Comment:
Does it matter which directory this tool is run from?

Reply:
No.